src/azul/chalice.py

```diff
@@ -64,6 +64,8 @@ class GoneError(ChaliceViewError):
 
 
 class AzulChaliceApp(Chalice):
+    # https://github.com/DataBiosphere/azul-chalice/blob/8d296f4cb1a7268300818cf53fc63d9cf310d471/chalice/constants.py#L57
+    DEFAULT_HANDLER_NAME = 'api_handler'
```

Comment:
Is the variable acceptable, or should it be inlined? I think the former given our discussions but just making sure.

Reply:
The point of referencing the constant in Chalice was that if the constant changes in a future release we wouldn't have to do anything. You're approach puts that on its head. Why did you deviate from what we'd already written together?